apps/frontend/src/pages/people/[username].tsx

```diff
@@ -275,7 +290,7 @@ const UserProfilePage: React.FC = () => {
     };
 
     fetchConnectionAndTapInfo();
-  }, [username, router]);
+  }, [username, router, user]);
```

Comment:
Added user dependency for when `user.userData.PullPSI` is updated by `refreshMessages`. When the user is updated it means refreshPSI was run, and we need to pick up the verified intersection so it can automatically populate the page.